src/op_mode/scan_localhost.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
```

Comment:
Why use two different script files if the logic is the same, just some arguments are different?